lib/workouts.js

```diff
@@ -1,157 +1,124 @@
 export const workouts = [
```

Comment:
Wow, a lot of content! We had Chat GPT generate more content for our project so that the app would be a bit more interesting. :wink: